feat(scaffolder-gitlab): add autoMerge input to publish:gitlab:merge-request action (#34947)
backstage/backstage: .changeset/gitlab-mr-auto-merge.md

---
'@backstage/plugin-scaffolder-backend-module-gitlab': patch
---

> Added an `autoMerge` boolean input to the `publish:gitlab:merge-request` scaffolder action. When set to `true`, the merge request is automatically merged once its pipeline succeeds, using GitLab's auto-merge feature.

Reviewer: ```suggestion
Added an `autoMerge` boolean input to the `publish:gitlab:merge-request` scaffolder action. When set to `true`, the merge request is automatically merged once all merge checks succeed, using GitLab's auto-merge feature.
```
